modules/programs/htop.nix

```diff
@@ -155,6 +182,38 @@ in {
       '';
     };
 
+    screens = mkOption {
+      type = hm.types.listOrDagOf (types.submodule ({ dagName, ... }: {
```

Comment:
The `listOrDagOf` type is deprecated and shouldn't be used. We really should remove it altogether soon.

So I would suggest to either use `listOf` or `dagOf`.